lib/ahoy/tracker.rb

```diff
@@ -176,11 +177,11 @@ def set_cookie(name, value, duration = nil, use_domain = true)
       cookie[:expires] = duration.from_now if duration
       domain = Ahoy.cookie_domain
       cookie[:domain] = domain if domain && use_domain
-      request.cookie_jar[name] = cookie
+      request.try { |r| r.cookie_jar[name] = cookie }
```

Comment:
May be a personal preference, but not a fan of the block form of `try`. How about `if request` instead (could even make it part of the `Ahoy.cookies` check above)?